<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPizaModelBuilder</class>
 <widget class="QDialog" name="DlgPizaModelBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Levels</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbLevels">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>40</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>68</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pillars</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbPillars">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>70</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>108</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sbHeight">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>105</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <double>4.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPreview">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>242</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>136</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Radius</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sbRadius">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>135</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <double>4.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgPizaModelBuilder</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgPizaModelBuilder</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
